zeroize/build.rs

```diff
@@ -43,7 +43,17 @@ mod linux {
             .unwrap();
 
         if !output.status.success() {
-            panic!("/usr/bin/ldd --version exited with error: {:?}", output);
+            let stderr = String::from_utf8(output.stderr).unwrap();
```

Comment:
My personal preference here would be to not further complicate the logic, and just change `get_glibc_version()` to return an `Option<String>`, returning `None` in this case.

Reply:
done